worker/README.md

```diff
@@ -1,3 +1,8 @@
+# DEPRECATED
```

Comment:
can you change to someting like 'for a more complete module see modbusdb ....

depracated is not correct here, using modbusdb is another option, it has some benefits over the worker, but it is also another dependency some users will opt to avoid

Reply:
changed